Tasks/DotNetCoreInstaller/task.json

```diff
@@ -0,0 +1,67 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "DotNetCoreInstaller",
+    "friendlyName": ".Net Core tool Installer (PREVIEW)",
+    "description": "Finds or Downloads and caches specified version spec of .Net Core SDK/runtime and adds it to the PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "preview": true,
+    "satisfies": ["DotNetCore"],
```

Comment:
There is not existing demand for dotnet core. Talked to kalyan and this name is fine.